src/main/java/net/pms/network/webinterfaceserver/handlers/BrowseHandler.java

```diff
@@ -463,7 +484,8 @@ private void addMediaLibraryFolderToFrontPage(
 		String headingKey,
 		String hasFolderVarName,
 		String childrenVarName,
-		HttpExchange t
+		HttpExchange t,
+		final HashMap<String, Object> mustacheVars
```

Comment:
it is use lines 517, 526 and 529.

Before, the `mustacheVars` map was declared on the class, so it was common to all request.
That's mean if 2 requests was served on the same time, they will fill the same `mustacheVars` object -> bug on result shown on the webpage.
That bug was not relevant because much of time, no one have 2 users browsing the page on the same time, and the server is constructing the result at the same time. But it was a bug.

The `mustacheVars` is now created on the request itself and passed to other function that need to write on it.

Reply:
@SurfaceS thanks for explaining